src/components/task-step/ends.cjsx

```diff
@@ -26,16 +30,86 @@ PracticeEnd = React.createClass
     </div>
 
 HomeworkEnd = React.createClass
-  render: ->
+
+  _addListener: ->
+    TaskStepStore.addChangeListener(@_update)
+
+  _removeListener: ->
+    TaskStepStore.removeChangeListener(@_update)
+
+  componentWillMount:   -> @_addListener()
+  componentWillUnmount: -> @_removeListener()
+  componentWillUpdate: -> @_removeListener()
+  componentDidUpdate:  -> @_addListener()
+  # override focus from within task steps to focus on first to do question
+  componentDidMount: -> document.querySelector('.task-step textarea')?.focus()
```

Comment:
that is happening, on each of the component's mount.  if i leave this line out right now, it'll focus on the last textarea that mounts, the last one that appears on the page